src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2039,6 +2039,30 @@ SpinelNCPInstance::property_set_value(
 				);
 			}
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_IPv6MeshLocalPrefix)) {
+			struct in6_addr address = any_to_ipv6(value);
+			char address_str[INET6_ADDRSTRLEN] = "::";
+
+			inet_ntop(AF_INET6, (const void *)&address, address_str, sizeof(address_str));
+
+			fprintf(stderr, "Using prefix \"%s\"\n", address_str);
```

Comment:
How about removing the log here?
The mesh-local prefix change will be logged (using `syslog`) on the spinel `VALUE_IS` reply for the `VALUE_SET` command. This happens from `SpinelNCPInstance::handle_ncp_spinel_value_is()` call here: https://github.com/openthread/wpantund/blob/master/src/ncp-spinel/SpinelNCPInstance.cpp#L2599

Reply:
Make sense. Updated.